docs/en/stack/ml/dataframes.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[ml-dataframes]]
+=== Data frames
```

Comment:
After https://github.com/elastic/docs/pull/902 is merged, this can be changed to:

```suggestion
=== {dataframes-cap}
```

Reply:
I've already changed it. Maybe that's why the check failed? In my local, the doc build was successful.